packages/board-server/tests/run-board.ts

```diff
@@ -139,6 +141,30 @@ const runStateStore = {
   async saveReanimationState(user: string, state: any) {
     return JSON.stringify(state);
   },
+  getBoardServerCorsConfig: function (): Promise<BoardServerCorsConfig | undefined> {
```

Comment:
See above. If we have `BoardServerStore`, then we don't need to reimplement all of these.